Polyfrost/Chatting
# use regex for pms

## Description
uses the same regex hytils uses for pms instead of the startsWith condition
the tab worked fine when I tested it but it didn't work in dev for some reason in dev even current master didn't work
didn't actually get to test the stash part bc idk how to trigger that but it should work :)

## Related Issue(s)

Fixes #11 (I think)

## How to test
1. Make sure chat tabs are on
2. Somehow cause items to go to your stash (idk how)
3. See if the messages are in the PMs tab

## Release Notes

```release-note
Fix SkyBlock stash pickups showing up in the PM tab.
```

## Documentation

No.
